x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/utils.test.ts

```diff
@@ -36,7 +36,7 @@ describe('utils', () => {
     test('should work with a full data set', () => {
       const fullRule = getResult();
       const rule = transformAlertToRule(fullRule);
-      const expected: OutputRuleAlertRest = {
+      const expected: Omit<OutputRuleAlertRest, 'throttle'> = {
```

Comment:
I'm still not sure why the omit is needed? Can't you just add `throttle` to the expected output? Since that is what is expected as the output?